wrims-core/src/main/java/wrimsv2/evaluator/DssOperation.java

```diff
@@ -305,6 +313,8 @@ public static void writeInitDvarAliasToDSS() {
 		while(iterator.hasNext()){
 			String initName=(String)iterator.next();
 			DssDataSet dds=DataTimeSeries.dvAliasInit.get(initName);
+			//TODO: DWR REVIEW MERGE
```

Comment:
@hxiedwr conflicting changes.

Reply:
[DssOperation.zip](https://github.com/user-attachments/files/17720327/DssOperation.zip)
I modified the DssOperation.java file to work with WRIMS_DevOps branch. I didn't code it in the WRIMS package so there might be some issues. Even if so, I don't think the effort to fix them would be major.

Reply:
Great! Thanks, I'll get this merged in now.

Reply:
I am not sure if you have any other issue with merging master into wrims_devops branch. If there is no Java code compilation errors, most likely the code merged works

Reply:
Everything is now compiling successfully with your updated DssOperation file, I'll go ahead and merge this pr into devops and it will be included in the next installer for testing.